udata_front/theme/gouvfr/datagouv-components/src/api/granularity.ts

```diff
@@ -0,0 +1,16 @@
+import axios from "axios";
+import { api_root_absolute } from "../plugins/api";
+
+export async function fetchGranularities() {
```

Comment:
I think we should add a variable containing the Promise to check it before doing the request (like https://github.com/Jorek57/udata-front/blob/feat/informations-component/udata_front/theme/gouvfr/assets/js/api/licenses.js#L6)